pywb/static/wombat.js

```diff
@@ -3663,13 +3735,12 @@ var _WBWombat = function($wbwindow, wbinfo) {
             override_func_first_arg_proxy_to_obj($wbwindow.Node, "contains");
             override_func_first_arg_proxy_to_obj($wbwindow.Document, "createTreeWalker");
 
-            override_func_this_proxy_to_obj($wbwindow, "setTimeout");
-            override_func_this_proxy_to_obj($wbwindow, "setInterval");
             override_func_this_proxy_to_obj($wbwindow, "getComputedStyle", $wbwindow);
             //override_func_this_proxy_to_obj($wbwindow.EventTarget, "addEventListener");
             //override_func_this_proxy_to_obj($wbwindow.EventTarget, "removeEventListener");
 
             override_apply_func($wbwindow);
+            initTimeoutIntervalOverrides($wbwindow);
```

Comment:
To avoid duplication, lets just run the same code for setTimeout and setInterval:
```
initTimeoutIntervalOverrides($wbwindow, "setTimeout");
initTimeoutIntervalOverrides($wbwindow, "setInterval");
```